### PR TITLE
Add instructions to run locally

### PR DESCRIPTION
**Pull request by exu3**
This PR adds appropriate `scripts` to the package.json to be able to run the site locally in addition to instructions in the README.

### CODE REVIEW

**Comment by bellesea** [comment]
> This PR adds appropriate `scripts` to the package.json to be able to run the site locally in addition to instructions in the README.

Thanks Ella!

File: README.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
 
 [Hack Club](https://hackclub.com/)'s [theme](https://theme.hackclub.com) implemented as CSS.
 
+## Development
```
Comment (sampoder) [inline README.md:5]: These aren't exactly going to help someone develop as it pulls styles from the internet, we could change how this works but for now this isn't going to help anyone

Comment (exu3) [reply README.md:5]: Ah ok, I realized that something was off when I was experimenting with theme.css.

How does one dev on the site?

**Review by sampoder — APPROVED** [review]
I wrote the code that makes these dev instructions work so im going to merge this, thanks so much Ella!

**Comment by sampoder** [comment]
oh side note, could you test those dev instructions to see if my code works on machines that aren't mine? Thanks again Ella

**Comment by bellesea** [comment]
would this be necessary?
`        <a class="no-styles-a" href="/theme.css"
          >CSS File</a
        >`
        
     instead of
`        <a class="no-styles-a" href="https://css.hackclub.com/theme.css"
          >CSS File</a
        >`

**Pull request merged**